app/views/documents/show/_withdrawn_notice.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "govuk_publishing_components/components/notice", {
+  title: I18n.t("documents.show.withdrawn.title",
+                document_type: @document.document_type.label.downcase,
+                withdrawn_date: @edition.status .created_at.strftime("%d %B %Y"))
+} do %>
+  <%= @edition.status.details.public_explanation %>
+  <p>
```

Comment:
As discussed, we can replace these two paragraphs with `<hr class="govuk-section-break--m">` (or `--l` modifier if we need more than space).

Reply:
Super, thank you 🙏

Reply:
As discussed, this element is being overridden so we're going to stick with two `<p>` tags for now.  @alex-ju to add to the backlog.

Reply:
It's not working within govspeak at the moment because of a CSS overwrite in govspeak, I'll add this to a card to look into clashes between govspeak styles and govuk-frontend.